src/Resources/config/ip_traceable.xml

```diff
@@ -12,11 +12,14 @@
             <call method="setAnnotationReader">
                 <argument type="service" id=".stof_doctrine_extensions.reader" on-invalid="ignore" />
             </call>
+            <call method="setIpAddressProvider">
+                <argument type="service" id="stof_doctrine_extensions.tool.ip_address_provider" on-invalid="ignore" />
```

Comment:
```suggestion
                <argument type="service" id="stof_doctrine_extensions.tool.ip_address_provider" />
```

That service is always defined in the bundle, so we don't need this `on-invalid` behavior. We can keep the default one which is failing if the service got removed for some reason.